changelog/db.changelog-1.37.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="Leigh Godson" id="1">
+    <tagDatabase tag="v1.37.0" />
+  </changeSet>
+  <changeSet id="2" author="Leigh Godson">
+    <modifyDataType tableName="delinkedCalculation" columnName="applicationId" newDataType="INTEGER"/>
+    <addColumn tableName="delinkedCalculation">
+      <column name="updated" type="TIMESTAMP"/>
+    </addColumn>
+  </changeSet>
+  <changeSet id="3" author="Leigh Godson">
+    <createTable tableName="d365">
+      <column name="d365Id" type="BIGINT" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="paymentReference" type="VARCHAR(30)">
+        <constraints nullable="false"/>
+      </column>
+      <column name="calculationId" type="INTEGER"/>
+      <column name="paymentPeriod" type="VARCHAR(200)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="paymentAmount" type="DECIMAL">
+        <constraints nullable="false"/>
+      </column>
+      <column name="transactionDate" type="DATE">
+        <constraints nullable="false"/>
+      </column>
+      <column name="datePublished" type="DATE">
+        <constraints nullable="true"/>
+      </column>
+      <column name="startPublish" type="DATE">
+        <constraints nullable="true"/>
+      </column>
+      <column name="completePublish" type="DATE">
+        <constraints nullable="true"/>
+      </column>
+      <column name="lastProcessAttempt" type="DATE">
```

Comment:
I think DATE and BIGINT are fine as types, but worth just verifying if we've used them before, and if not, we are careful with the Dev deployment and make sure liquibase accepts it there.

Reply:
They are both in the allowed mappings but I think for consistency I will change DATE to TIMESTAMP and BIGINT to INTEGER